src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1185,30 +1207,44 @@
 helps['network nic'] = """
     type: group
     short-summary: Manage network interfaces.
+    long-summary: >
+        To learn more about network interfaces in Azure visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-network-interface
 """
 
 helps['network nic show-effective-route-table'] = """
     type: command
     short-summary: Show all route tables applied to a network interface.
+    long-summary: >
+        To learn more about how to troubleshoot using effective route tables visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-routes-troubleshoot-portal#using-effective-routes-to-troubleshoot-vm-traffic-flow
 """
 
 helps['network nic list-effective-nsg'] = """
     type: command
-    short-summary: List all network security groups applied to a network interface.
+    short-summary: List all effective network security groups applied to a network interface.
```

Comment:
If `effective` here means `in effect`, use the clearer wording. `Effective` is ambiguous when looking at help, especially relating to diagnostic ("effective troubleshooting of X" vs. "troubleshooting effective X" is easily confused.)